cypress/integration/device/device-details.spec.ts

```diff
@@ -55,6 +60,34 @@ describe('Test device details page', () => {
     cy.wait('@get-auditlog').its('response.statusCode').should('eq', 200)
     cy.wait('@get-features').its('response.statusCode').should('eq', 200)
 
+    // Out-of-band Power Actions
+    cy.contains('Power On').click()
```

Comment:
these look like good candidates for a for loop

Reply:
fixed